docs/development/_sidebar.md

```diff
@@ -8,11 +8,17 @@
 
   - [Getting started](development/getting-started.md)
 
-  - [Hot reloading web](development/hot-reloading-web.md)
+  - Guides
 
-  - [RFCs (Request For Comments)](https://iver-wharf.github.io/rfcs)
+    - [Changelog](development/changelogs/)
 
-  - [Debugging in GoLand](development/debugging-in-goland.md)
+    - [RFCs (Request For Comments)](https://iver-wharf.github.io/rfcs)
+
+  - Tips and tricks
+
+    - [Hot reloading web](development/hot-reloading-web.md)
+
+    - [Debugging in GoLand](development/debugging-in-goland.md)
```

Comment:
I think, personally, that these two still fall into the category of Guides.
Are they separated because the changelog and RFCs are considered more or less obligatory, while these are more optional QoL things?

Reply:
I just wanted to group them somehow because the list became difficult to read. But then after I added the Changelog section it did get a little easier, so we can group them back together.

My use of "Guides" vs "Tips and tricks" were perhaps poor choice of words